publication-commons/src/main/java/no/unit/nva/publication/service/impl/TicketService.java

```diff
@@ -132,6 +132,7 @@ protected TicketEntry completeTicket(TicketEntry ticketEntry) throws ApiGatewayE
         var publication = resourceService.getPublicationByIdentifier(ticketEntry.getResourceIdentifier());
         var existingTicket =
             attempt(() -> fetchTicketByIdentifier(ticketEntry.getIdentifier()))
+                .or(() -> fetchByResourceIdentifierForLegacyDoiRequestsAndPublishingRequests(ticketEntry))
```

Comment:
Current doi request handlers do not support ticket identifiers and work based on the fact that doi requests are unique for each publication.